Source/Core/UICommon/DiscordPresence.cpp

```diff
@@ -89,88 +88,23 @@ void HandleDiscordJoin(const char* join_secret)
-      "GM8",  // GM8E01: Metroid Prime
-      "G2M",  // G2ME01: Metroid Prime 2: Echoes
-      "RM3",  // RM3E01: Metroid Prime 3: Corruption
-      "R3M",  // R3ME01: Metroid Prime: Trilogy
-      "SMN",  // SMNE01: New Super Mario Bros. Wii
-      "G8M",  // G8ME01: Paper Mario: The Thousand-Year Door
-      "GPI",  // GPIE01: Pikmin (GC)
-      "R9I",  // R9IE01: Pikmin (Wii)
-      "GPV",  // GPVE01: Pikmin 2 (GC)
-      "R92",  // R92E01: Pikmin 2 (Wii)
-      "GC6",  // GC6E01: Pokemon Colosseum
-      "GXX",  // GXXE01: Pokemon XD: Gale of Darkness
-      "GBI",  // GBIE08: Resident Evil
-      "GHA",  // GHAE08: Resident Evil 2
-      "GLE",  // GLEE08: Resident Evil 3: Nemesis
-      "G4B",  // G4BE08: Resident Evil 4
-      "GSN",  // GSNE8P: Sonic Adventure 2: Battle
-      "GXS",  // GXSE8P: Sonic Adventure DX: Director's Cut
-      "SNC",  // SNCE8P: Sonic Colors
-      "G9S",  // G9SE8P: Sonic Heroes
-      "GRS",  // GRSEAF: SoulCalibur II
-      "RSL",  // RSLEAF: SoulCalibur Legends
-      "GK2",  // GK2E52: Spider-Man 2
-      "GQP",  // GQPE78: SpongeBob SquarePants: Battle for Bikini Bottom
-      "SVM",  // SVME01: Super Mario All-Stars: 25th Anniversary Edition
-      "RMG",  // RMGE01: Super Mario Galaxy
-      "SB4",  // SB4E01: Super Mario Galaxy 2
-      "G4Q",  // G4QE01: Super Mario Strikers
-      "GMS",  // GMSE01: Super Mario Sunshine
-      "GMB",  // GMBE8P: Super Monkey Ball
-      "GM2",  // GM2E8P: Super Monkey Ball 2
-      "R8P",  // R8PE01: Super Paper Mario
-      "RSB",  // RSBE01: Super Smash Bros. Brawl
-      "GAL",  // GALE01: Super Smash Bros. Melee
-      "PZL",  // PZLE01: The Legend of Zelda: Collector's Edition
-      "G4S",  // G4SE01: The Legend of Zelda: Four Swords Adventures
-      "D43",  // D43E01: The Legend of Zelda: Ocarina of Time / Master Quest
-      "SOU",  // SOUE01: The Legend of Zelda: Skyward Sword
-      "GZL",  // GZLE01: The Legend of Zelda: The Wind Waker
-      "GZ2",  // GZ2E01: The Legend of Zelda: Twilight Princess (GC)
-      "RZD",  // RZDE01: The Legend of Zelda: Twilight Princess (Wii)
-      "GHQ",  // GHQE7D: The Simpsons: Hit & Run
-      "RSP",  // RSPE01: Wii Sports
-      "RZT",  // RZTE01: Wii Sports Resort
-      "SX4",  // SX4E01: Xenoblade Chronicles
-  };
-
-  std::string region_neutral_gameid = gameid.substr(0, 3);
-  if (REGISTERED_GAMES.count(region_neutral_gameid) != 0)
-  {
-    // Discord asset keys can only be lowercase.
-    Common::ToLower(&region_neutral_gameid);
-    return "game_" + region_neutral_gameid;
-  }
-  return "";
+  DiscIO::Region region = SConfig::GetInstance().m_region;
+  bool isWii = SConfig::GetInstance().bWii;
+  std::string region_code = SConfig::GetInstance().GetGameTDBImageRegionCode(region, isWii);
+
+  constexpr char cover_url[] = "https://art.gametdb.com/wii/cover/{}/{}.png";
+  std::string url = fmt::format(cover_url, region_code, SConfig::GetInstance().GetGameTDBID());
```

Comment:
```suggestion
  auto game_id = SConfig::GetInstance().GetGameTDBID()
  std::string url = fmt::format(cover_url, region_code, game_id);
```

Reply:
Nitpicking, but it seems as creating a variable first is the standard way in this code, instead of calling it directly in the function.